### DISCUSSION TITLE
Subunits of an Organisation

### DISCUSSION THREAD

**Discussion by giorgialodi** [eprocurementontology/eprocurementontology#498] [General]
In a ANAC's dataset that we are mapping in the context of the BDTI pilot, there is data regarding subunits of the organisation that plays the role of buyer. 
It seems to me that in the OWL files there are no definitions to model this data.

There are two possible solutions, if we want to target this issue (at the time being that data on the subunits is not at all mapped in our pilot):

1) we declare somewhere the alignment (subClassOf) of our http://data.europa.eu/a4g/ontology#Organisation to org:Organization. In this case we could use org:Organization properties such as hasUnit and then use OrganizationalUnit (which is an Organization) to specify these subunits

2) we add in our namespace the modelling of subunits as it is done in org:organization and then we align all our classes and properties to org ontology.

Do you think it makes sense to model subunits of organisations in our domain?

**Comment by paulakeen** [comment]
If we align to the w3c Org base class Organization....are we not indirectly aligning to everything that inherits from it, too? 

In principle, I do not see the interest of knowning which is the parent organisation of a subunit that is a Buyer. The relevant information is who is the buyer, regardless of whether it is a sub-unit or not. No?

**Comment by JachymHercher** [comment]
> In principle, I do not see the interest of knowning which is the parent organisation of a subunit that is a Buyer. The relevant information is who is the buyer, regardless of whether it is a sub-unit or not. No?

I've seen this request come up reasonably often. In eForms, it led to a very simplistic solution of Organisation Part Name (BT-16). I think some sort of nesting of organisations would be better. 

A basic example of when this may be useful is the European Commission. Legally, it is a single contracting authority, so e.g. eForms would treat it as a single buyer, but many users of the data would be interested in procurement at the directorate general level.

**Comment by SellittoGiampaolo** [comment]
Not quite so, Paula...
The legal person that is the Buyer remains the parent organization (The Fiscal Code is the one of the parent organization), while the sub-units can define where the service or the good or the work is located (consumed) so both pieces of information are important, especially in MS where regions are important. 
Take as an example a Big central administration with local units that buy services, if we monitor procurement using only the central administration instead of the local units the indicators can vary a lot

**Comment by giorgialodi** [comment]
> If we align to the w3c Org base class Organization....are we not indirectly aligning to everything that inherits from it, too?

Yes we may inherit what we need in our domain as the example above of the organizational units 
In any case, if we want to consider this to be very lightweight, we can go for solution 2 above and then align to org. But at least we define in our domain just the necessary and no more than that. For instance in ANAC's dataset regarding Lot and Procedure I have an ID of the organizational unit (that could be used to form the URI) and the name, and an implicit link to the main buyer (such as a Region). That's it.  
For very big administrations as the examples above of @SellittoGiampaolo and @JachymHercher it might indeed be useful to consider subunits of organisations.

**Comment by jpmckinney** [comment]
> In principle, I do not see the interest of knowning which is the parent organisation of a subunit that is a Buyer. The relevant information is who is the buyer, regardless of whether it is a sub-unit or not. No?

We have seen this come up regularly across jurisdictions. It's quite common for a user to want to aggregate procedures by parent organization, rather than by individual unit.

**Comment by ec-mcs** [comment]
There are various possibilities to get the information of the organizational structure. Please be careful of making the data model more complex. In order to achieve interoperability, ALL must implement it. So Jachyms idea to make use of BT-16 is a good one. Besides that, not everything must be encoded in the notices. MS could provide information about organizational structures that could be used to retrieve the necessary information.

**Comment by giorgialodi** [comment]
> There are various possibilities to get the information of the organizational structure. Please be careful of making the data model more complex. In order to achieve interoperability, ALL must implement it. So Jachyms idea to make use of BT-16 is a good one. Besides that, not everything must be encoded in the notices. MS could provide information about organizational structures that could be used to retrieve the necessary information.

Honestly I do not see any complex model for subunits of organisations. We just need to consider a concept that models subunits of organisations that is an organisation itself (somehow represents the Organisation Part Name) and that has a relationship with the organisation (like hasSubunit - that represents somehow what @JachymHercher mentions as "nesting of organisations"). That's it.

**Comment by jpmckinney** [comment]
I think it's fine to describe a government's organizational hierarchy in a separate dataset, though very few governments do this, leading to third-party initiatives like https://github.com/okfn/publicbodies

As @giorgialodi and @JachymHercher describe, there are simple solutions that can serve the use cases in this thread without modelling all of government from the perspective of a procurement procedure :)

**Comment by paulakeen** [comment]
The WG decides to look into this at a later time and try to align with the W3C Organization so we can model the subunits by reusing what the W3C ORG ontology already proposed.

**Comment by andreea-pasare** [comment]
In the latest release version, ePO 3.0.1, we have the attribute epo:hasOrganisationUnit that can be used:

![image](https://user-images.githubusercontent.com/88081600/197235454-ccf812d1-64ca-4349-8534-0136a156f54d.png)

**Comment by giorgialodi** [comment]
Sorry @andreea-pasare but I don't think you can do  this modelling. Are you adding properties to org:Organisation? How is it possible? I think that you have to create your own Organisation class, add these properties and say that your organisation is a subclass of org:Organization. BTW: OrganisationUnit is already available as concept in the Org ontology and it is definitely not a Literal but an object! Please re-consider this overall modelling. @muricna

**Comment by giorgialodi** [comment]
Ah, also LegalFormType can be modelled using Org ontology. Therefore, it is important to understand which modelling approach to follow in the presence of an ontology that is outside the control of the EU Commission. If you decide to re-use directly org:Organisation, please use it in its main elements. If you believe that there are specific properties that are to be added (with specific restrictions too) in your domain then it is safer to define your own class that is in any case aligned with the external concept.

**Comment by andreea-pasare** [comment]
@giorgialodi, a proper alignment methodology with Core Vocabularies is under discussion. In addition, it was not clear how the alignment to other ontologies should be treated. We will address this when an agreement with Core Vocabulary team is reached.

As far as organisation subunit is concerned, the current implementation fulfills the requirements of the eForms as depicted in the image below:
![image](https://user-images.githubusercontent.com/88081600/197534295-f794abf0-c339-4b86-9c80-fd2360b4e0f4.png)

We will also take into consideration your comments about adopting Org ontology. Re-opening this issue until we come to an agreement within the working group.

**Comment by giorgialodi** [comment]
@andreea-pasare In the UML diagram you show in this thread you already adopt org:Organization since the main class comes from that ontology. I understand that the proposed modelling is complaint with eForms but you can model that excel file in different ways. In org ontology there is the concept of OrganisationalUnit (or SupportUnit) that is an organisation itself (IMHO, it is the best modelling approach considering the complexities of EU institutions) that has a name (also this modelling is compliant with the eForms excel file and it is definitely nicer than using a datatype property. From strings to codes :) ). BTW: Core Public Organisation Vocabulary reuses org:Organization including the OrganisationalUnit part which is what you are trying to model here.
